eng/pipelines/common/templates/runtimes/wasm-runtime-and-send-to-helix.yml

```diff
@@ -0,0 +1,88 @@
+parameters:
+  buildConfig: ''
+  archType: 'wasm'
+  osGroup: 'Browser'
+  osSubgroup: ''
+  container: ''
+  testGroup: ''
+  crossrootfsDir: ''
+  readyToRun: false
+  liveLibrariesBuildConfig: ''
+  crossgen2: false
+  compositeBuildMode: false
+  helixQueues: ''
+  stagedBuild: false
+  displayNameArgs: ''
+  runInUnloadableContext: false
+  runtimeVariant: ''
+  variables: {}
+  pool: ''
+  runtimeFlavor: 'mono'
+  runtimeFlavorDisplayName: 'Mono'
+  dependsOn: []
+  #arcade-specific parameters
+  condition: always()
+  continueOnError: false
+  displayName: ''
+  timeoutInMinutes: ''
+  enableMicrobuild: ''
+  gatherAssetManifests: false
+
+
+steps:
+  - script: $(Build.SourcesDirectory)/src/coreclr/build-test$(scriptExt) /p:LibrariesConfiguration=${{ parameters.buildConfig }} -skipnative  -skipstressdependencies -excludemonofailures os Browser wasm
+    displayName: Build Tests
+
+  #  Send tests to Helix
+  - template: /eng/pipelines/common/templates/runtimes/send-to-helix-step.yml
+    parameters:
+      displayName: Send tests to Helix
+      ${{ if eq(parameters.buildConfig, 'debug') }}:
+        buildConfig: 'Debug'
+      ${{ if eq(parameters.buildConfig, 'release') }}:
+        buildConfig: 'Release'
+      ${{ if eq(parameters.buildConfig, 'checked') }}:
+        buildConfig: 'Checked'
+      archType: ${{ parameters.archType }}
+      osGroup: ${{ parameters.osGroup }}
+      osSubgroup: ${{ parameters.osSubgroup}}
+      coreClrRepoRoot: $(Build.SourcesDirectory)/src/coreclr
+      runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}
+
+      ${{ if eq(variables['System.TeamProject'], 'public') }}:
```

Comment:
Is there a way were we could share all these by adding an extra template or something?

Reply:
Maybe; I am not sure what other template you mean for me to share these with?

Reply:
I meant, maybe having a template that is shared in run-tests-job and here so that we don't duplicate all the send-to-helix parameter logic in both places.

Reply:
To be resolved in follow up issue.